README.rst

```diff
@@ -49,9 +49,11 @@ Bergen *et al.* (2020), Generalizing RNA velocity to transient cell states throu
 
 Support
 ^^^^^^^
-Feel free to submit an `issue <https://github.com/theislab/scvelo/issues/new/choose>`_,
-start a new `discussion <https://github.com/theislab/scvelo/discussions>`_,
-or send us an `email <mailto:mail@scvelo.org>`_.
+Found a bug or would like to see a feature implemented? Feel free to submit an
```

Comment:
Could also be rephrased to _"Something is not working as expected ..."_ if we do not to use coding / software dev. terms.